tests/vite-app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "vite-app",
+  "private": true,
+  "version": "0.0.2",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "tokengen": "tsc util/generateExampleTokens.ts && node util/generateExampleTokens.js",
```

Comment:
We could use ts-node perhaps to directly run the ts file.
Saves adding the js to prettierignore and such :D